app/helpers/application_helper.rb

```diff
@@ -361,6 +361,15 @@ def get_user_access_token(user)
     end
   end
 
+   # Return a scope-limited access token that can be used in URLs (e.g. in urls passed to Morpheus)
+  def get_url_safe_access_token(user)
```

Comment:
It'd help to have a brief code comment on the difference between a URL safe access token and a TOTAT.